README.md

```diff
@@ -0,0 +1,50 @@
+# SSV Distributed Key Generation Spec
+
+## Introduction
+This repository contains spec of distributed key generation (DKG) used in SSV. Based on verifiable secret sharing, DKG allows a set of *n* operators to generate a  **BLS** key share for each operator, such that a BLS private key can only be used when threshold *t* out of *n* operators agree. 
+
+---
+
+## Functionalities
+Operations are initiated by an initiator, and executed by operators. An initiator can initiate:
+
+### Init
+When a init message is sent to the operators, all operators participate and run a DKG ceremony to create key shares. Each operator generates a result with a **proof** to show the correctness of execution. Proofs are referred to when an initiator asks for re-sign or reshare. The encrypted share can be found in a proof as it could be published to a smart contract.
```

Comment:
Can you explain in more details how this proof works?

Reply:
updated